in_toto/models/layout.py

```diff
@@ -140,8 +140,11 @@ def import_step_metadata_from_files_as_dict(self):
     """
     step_link_dict = {}
     for step in self.steps:
-      link = models__link.Link.read_from_file(step.name + '.link')
-      step_link_dict[step.name] = link
+      key_link_dict = {}
+      for keyid in step.pubkeys:
+        link = models__link.Link.read_from_file(step.name + '.' + keyid + '.link')
```

Comment:
Great, I suppose you found [this line](https://github.com/in-toto/in-toto/blob/develop/in_toto/runlib.py#L382) or located the relevant part in the [format specification mini language](https://docs.python.org/2/library/string.html#format-specification-mini-language)?

> The precision is a decimal number indicating how many digits should be displayed after the decimal point for a floating point value formatted with 'f' and 'F', or before and after the decimal point for a floating point value formatted with 'g' or 'G'. For non-number types the field indicates the maximum field size - in other words, how many characters will be used from the field content. The precision is not allowed for integer values.